Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/ui/location/ContactDiaryAddLocationViewModel.kt

```diff
@@ -31,25 +31,39 @@ class ContactDiaryLocationBottomSheetDialogViewModel @AssistedInject constructor
         ex.report(ExceptionCategory.INTERNAL, TAG)
     }
 
-    private val text = MutableStateFlow("")
+    val shouldClose = SingleLiveEvent<Unit>()
 
-    val isValid = text.map {
-        it.isNotEmpty() && it.length <= MAX_LOCATION_NAME_LENGTH
-    }.asLiveData()
+    private val _locationName = MutableStateFlow("")
+    val locationName: StateFlow<String> = _locationName
```

Comment:
You're right, we don't need them anymore. I wanted to get rid of the get for the formattedName because I realised fromatContactDiaryNameField wasn't doing much and I thought I'd need public access but then I forgot to remove it and revert to using only private members.

I'll remove the public values.